algorithms/ar-expsq/java/ExponentiationBySquaring.java

```diff
@@ -0,0 +1,31 @@
+class ExponentiationBySquaring {
```

Comment:
Codacy found an issue: [The utility class name 'ExponentiationBySquaring' doesn't match '[A-Z][a-zA-Z0-9]+(Utils?|Helper|Constants)'](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6356801)